Reflections/Week_10_Reflection.md

```diff
@@ -0,0 +1,14 @@
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  - Got the most basic linear actuator made to get an understanding of what I need. 
+  - Updated some of the files throughout the project
+  - Have a longboard to test if the system currently has the power to move it as well.
+3. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  - I found the parts I requested was mislabled on the amazon listing. The part needed was shorted than expected. I have ordered a new part, but it will take time to come in, and so I have started to look into different ways to build what I need with what I have.
+5. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+  - See if the current system has the power to move the longboard. This could take about 2 hours to complete.
+    - Specific: Take the built system off the old board, and stransfer it too the new longboard. Run it and see if it struggles to move the longbaord or moves it without trouble.
+    - Measurable: Time the smaller board at how mlong it takes to get to the end of the length I have specified. Using the time it takes to determine if it struggles or not.
+    - Achievable: This can be done easily, but the difficulty might come from the length of the board difference, otherwise this is achievable.
```

Comment:
Will you be able to use the same linear actuator for different sized skateboards? Will you use two different lead screws?